scripts/Dockerfile

```
@@ -0,0 +1,16 @@
+FROM debian:testing
+
+ENV RISCV_DIR=/toolchain/riscv64-unknown-elf-gcc-8.3.0-2020.04.1-x86_64-linux-ubuntu14/bin
```

Won't this only needed during the build time? In that time probably we can change it to `ARG`
```suggestion
ARG RISCV_DIR=/toolchain/riscv64-unknown-elf-gcc-8.3.0-2020.04.1-x86_64-linux-ubuntu14/bin
```

---

This variable is listed by the README, but I'm not fully sure it's needed. I will double check it.

---

Correct, RISCV_DIR is not needed or used.